src/hm-admin/hm-admin.component.ts

```diff
@@ -1,11 +1,26 @@
-import { Component } from '@angular/core';
+import {Component, Input, OnInit} from '@angular/core';
+import {ConfigService} from './services/config/config.service';
 
 @Component({
   moduleId: module.id,
   selector: 'hm-admin-app',
   templateUrl: 'hm-admin.component.html',
-  styleUrls: ['hm-admin.component.css']
+  styleUrls: ['hm-admin.component.css'],
+  providers: [ConfigService]
 })
-export class HmAdminComponent {
+export class HmAdminComponent implements OnInit {
+  @Input('config')
+  config: any;
+
+
+  constructor(private configService:ConfigService) {
+    this.configService = configService;
```

Comment:
with DI and magical declaration of typescript this line is useless.
```constructor(private configService:ConfigService) {```
The `private` will do the job for you